### ISSUE TITLE
ldpd status ? 

### ISSUE THREAD

**Issue by scramatte** [rwestphal/quagga-public#1]
Hello, 

Can you told me what is ldpd daemon status ? Does it usable in production ? 

Thank you

**Comment by rwestphal** [comment]
Hi,

I think the ldpd daemon is relatively stable, but certainly not ready for use in production. If you want to help, please try the code yourself and provide a small feedback to me (bug reports would be nice too).

To build quagga with MPLS/LDP support you'll need libnetlink (provided by iproute-dev on Debian).

To build and install, just type:
$ ./configure --enable-mpls --enable-ldpd [your flags]
$ make
# make install (or copy the binaries and conf files by hand)

If you want to test the ldpd daemon on a non-mpls enabled kernel, just pass '--enable-mpls=null' to the configure script.

[]s

**Comment by scramatte** [comment]
Thank you for the answer  and sorry for late reply.  Unfortunately I haven't got  time to test it yet :(
Hope to be able to do it soon.

**Comment by mina790124** [comment]
HI!
I install MPLS-Linux(https://github.com/i-maravic/MPLS-Linux)  and 
mpls ldpd daemon  kernel(https://github.com/rwestphal/quagga-public).
How to run and set ldpd from ldpd.conf.sample??

 ldpd.conf.sample>>
                  ! -_\- ldp -_-
                  ! LDPd sample configuration file
                  hostname ldpd
                  password zebra
                  log stdout
                  interface eth0
                  mpls ip

**Comment by MHZarei** [comment]
Hi
I ran LDP and OSPF on Linux Debian wheezy. Also in installed iproute-dev on it.
How I can watch MPLS tag in Linux route table and validate MPLS work on my machin?  IP ("ip route") command don't have any options for MPLS!

**Comment by rwestphal** [comment]
Sorry but I don't support this repository anymore, you are by yourself.

I suggest you to try OpenBSD, it supports MPLS out of the box and it's much
superior in this area.

[]s

2015-11-03 11:17 GMT-02:00 M.H.Zarei notifications@github.com:

> Hi
> I ran LDP and OSPF on Linux Debian wheezy. Also in installed iproute-dev
> on it.
> How I can watch MPLS tag in Linux route table and validate MPLS work on my
> machin? IP ("ip route") command don't have any options for MPLS!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/rwestphal/quagga-public/issues/1#issuecomment-153348740
> .

## 

Renato Westphal

**Comment by mehrshadsa** [comment]
Hi

Can i run mpls ldpd in centos linux ? and connect to quagga?
please help me

Thanks
Mehrshad shahmohammadi

**Comment by MHZarei** [comment]
Yes, You can run MPLS ldpd in Linux base systems. But just LDP protocol for labeling work in Linux.

**Comment by rwestphal** [comment]
Yes. I'm working on a new port of ldpd, please check it out:
https://github.com/rwestphal/quagga-ldpd/commits/ldpd-port
https://github.com/rwestphal/quagga-ldpd/wiki/ldpd-basic-test-setup
https://github.com/rwestphal/openbsd-ldpd/commits/renato-2016

**Comment by mehrshadsa** [comment]
dear MHZarei and rwestphal 

Thank you so much.

can i implement ldp with following web site?

https://github.com/rwestphal/quagga-ldpd/commits/ldpd-port
https://github.com/rwestphal/quagga-ldpd/wiki/ldpd-basic-test-setup
https://github.com/rwestphal/openbsd-ldpd/commits/renato-2016
